docs/modules/ROOT/pages/explanations/decisions/kafka.adoc

```diff
@@ -0,0 +1,92 @@
+= VSHN Managed Kafka
+
+== Problem
+
+Kafka is a very complex software.
```

Comment:
I'd add a few words that it's not "a single software", but a whole suite of different applications (https://strimzi.io/docs/operators/latest/overview#kafka-components_str)